Omise.Net/RequestManager.cs

```diff
@@ -13,6 +13,7 @@ public class RequestManager : IRequestManager
         private string apiUrlBase;
         private string apiKey;
         private string encodedCredentials;
+        private readonly string clientVersion = "1.0.5";

```

Comment:
Would it be possible to read this value from somewhere else or is it too complicated? But then you need to be careful to keep this version in sync in all the other places.